contrib/pg_tde/documentation/docs/yum.md

```diff
@@ -32,15 +28,15 @@ You need the `percona-release` repository management tool that enables the desir
 
     The `pg_tde` {{release}} extension is a part of the `percona-postgresql17` package. If you installed a previous version of `pg_tde` from the `percona-pg_tde_17` package, do the following:
 
-    * Drop the extension using the `DROP EXTENSION` with `CASCADE` command.
+    1. Drop the extension using the `DROP EXTENSION` with `CASCADE` command.
 
-       <i warning>:material-alert: Warning:</i> The use of the `CASCADE` parameter deletes all tables that were created in the database with `pg_tde` enabled and also all dependencies upon the encrypted table (e.g. foreign keys in a non-encrypted table used in the encrypted one).    
+       The use of the `CASCADE` parameter deletes all tables that were created in the database with `pg_tde` enabled and also all dependencies upon the encrypted table (e.g. foreign keys in a non-encrypted table used in the encrypted one).    
 
        ```sql
        DROP EXTENSION pg_tde CASCADE
        ```
 
-    * Uninstall the `percona-pg_tde_17` package.  
+    2. Uninstall the `percona-pg_tde_17` package.  
```

Comment:
Shouldn't we also mention that they should remove the pre-load library? Plus isn't there a risk of there being encrypted objects like sequences and indexes left over which uses the SMGR but not the AM? Do we need to add a way to heck for that? Plus encrypted WAL.

In general uninstalling TDE safely is no easy task.